src/skin/tooltips.cpp

```diff
@@ -723,18 +751,34 @@ void Tooltips::addStandardTooltips() {
             << tr("Next or Previous")
             << tr("Switch to either the next or previous effect.");
 
+    add("EffectSlot_metaknob")
+            << tr("Meta Knob")
+            << tr("Controls linked parameters of this effect")
+            << QString("%1: %2").arg(rightClick, resetToDefault);
+
+    add("EffectSlot_focus")
+            << tr("Effect Focus Button")
+            << QString("%1: %2").arg(leftClick, tr("Focuses this effect."))
+            << QString("%1: %2").arg(rightClick, tr("Unfocuses this effect."))
+            << tr("Refer to the web page on the Mixxx wiki for your controller for more information.");
+
     add("EffectSlot_parameter")
             << tr("Effect Parameter")
             << tr("Adjusts a parameter of the effect.")
             << QString("%1: %2").arg(rightClick, resetToDefault);
 
     add("EffectSlot_parameter_link_type")
-            << tr("Super Knob Mode")
-            << tr("Set how linked effect parameters change when turning the Super Knob.");
+            << tr("Meta Knob Link")
+            << tr("Set how this parameter is linked to the effect's Meta Knob.")
+            << tr("Empty bar: not linked")
```

Comment:
@Be-ing This line ff. needs some fine tuning. While translating, i found them pretty hard to interpret (read: provide correct translations). Each line can be displayed for its own in the Transifex translation interface, so context can be rather vague.

We define ``bar`` [in our glossary](https://www.transifex.com/mixxx-dj-software/mixxxdj/glossary/en/?filter=bar) different. Instead of adding another definition, i`d describe the status of the multi-state button.

Tooltips tend to be lengthy already, so i don´t mind some extra chars.

Proposal:
``Inactive: Parameter not linked``
``Active: Parameter moves with Meta Knob``
``Left side active: Parameter moves with left half of Meta Knob``
``Right side active: Parameter moves with right half of Meta Knob``
``Left and right side active: Parameter moves across range with half of Meta Knob and back with the other half``

Thoughts?

Any plans to update the [controllerpicker menu](https://github.com/mixxxdj/mixxx/blob/master/src/controllers/controlpickermenu.cpp) as well?
Unfortunately we have to maintain [two sources](https://bugs.launchpad.net/mixxx/+bug/1403748) for strings :/